contracts/context-config/tests/sandbox.rs

```diff
@@ -427,18 +427,18 @@ async fn main() -> eyre::Result<()> {
         .call(contract.id(), "mutate")
         .args_json(Signed::new(
             &{
-                let kind = RequestKind::Context(ContextRequest {
+                let kind = RequestKind::Context(ContextRequest::new(
                     context_id,
-                    kind: ContextRequestKind::UpdateApplication {
-                        application: Application {
-                            id: new_application_id,
-                            blob: new_blob_id,
-                            size: 0,
-                            source: Default::default(),
-                            metadata: Default::default(),
-                        },
+                    ContextRequestKind::UpdateApplication {
+                        application: Application::new(
+                            new_application_id,
+                            new_blob_id,
+                            0,
+                            Default::default(),
+                            Default::default(),
+                        ),
```

Comment:
The point is that anything exposed externally needs to be thought about. If something is truly exhaustive, that's fine and it can be marked as such... if not exhaustive then best practice is to mark it as `non_exhaustive` and that way consumers of the code know not to rely on it. One of the most important aspects of this is that it's then not possible to construct via struct syntax, although it is still possible to access public fields. As we stabilise the public interface we need to be thinking about these things and working in a semver-compatible manner, which is what this assists.